src/main/java/decodes/tsdb/TsdbAppTemplate.java

```diff
@@ -384,24 +369,14 @@ public void tryConnect()
     }
 
     /**
-     * @param afn
-     * @param ex
+     * @param afn auth filename
+     * @param ex exception
      */
     protected void authFileEx(String afn, Exception ex)
     {
-        String msg = "Cannot read DB auth from file '" + afn + "': " + ex;
-        System.err.println(msg);
-        Logger.instance().failure(msg);
-try { throw new Exception(""); } catch (Exception ex2) { ex2.printStackTrace(); }
+        log.error("Cannot read DB auth from file '{}'", afn ,ex);
```

Comment:
I believe this form doesn't work and the .atError().setCause(ex).log(...) needs to be used.
But easy to test with a bad auth file.